core/src/main/scala/org/http4s/EntityDecoder.scala

```diff
@@ -156,12 +156,18 @@ trait EntityDecoderInstances {
   implicit def binary[F[_]: Sync]: EntityDecoder[F, Chunk[Byte]] =
     EntityDecoder.decodeBy(MediaRange.`*/*`)(collectBinary[F])
 
+  implicit def byteArrayDecoder[F[_]: Sync]: EntityDecoder[F, Array[Byte]] =
```

Comment:
I think this one was requested before and I was surprised we didn't have it.  I don't like people using byte arrays, but people do.